src/components/AppMapDetailsObj.ts

```diff
@@ -415,4 +418,100 @@ export default class AppMapDetailsObj extends AppMapDetailsBase<MapMarkerObj | M
     }
     return "Normal";
   }
+  korokType() {
+    //if (!this.obj || !('korok_id' in this.obj)) {
+    //  return "NaK";
+    //}
+    return this.obj && this.obj.korok_type == 'Flower Trail';
+  }
+
+  findItemByHash(group: any[], links: any[], name: string): any {
+    let hashes = links.map(link => link.DestUnitHashId);
+    let out = group.filter(g => g.data.UnitConfigName == name && hashes.includes(g.hash_id));
+    if (out.length == 1) {
+      return out[0];
+    }
+    return undefined;
+  }
+
+  nextFlower(group: any[], flower: any): any {
+    let or = this.findItemByHash(group, flower.data.LinksToObj, "LinkTagOr");
+    if (!or) {
+      return undefined;
+    }
+    let lag = this.findItemByHash(group, or.data.LinksToObj, "SwitchTimeLag");
+    if (!lag) {
+      return undefined;
+    }
+    let and = this.findItemByHash(group, lag.data.LinksToObj, "LinkTagAnd");
+    if (!and) {
+      return undefined;
+    }
+    let p = this.findItemByHash(group, and.data.LinksToObj, "Obj_Plant_Korok_A_01");
+    return p;
+  }
+
+  isLastFlower(flower: any): boolean {
+    return flower.data['!Parameters'].IsLastKorokFlower;
+  }
+  korokIcon(icon: string, style: string, text: string = ""): L.DivIcon {
+    let html: string = `<div><i class="fa ${icon} korokicon" style="${style}"></i>${text}</div>`;
+    return L.divIcon({
+      className: '',
+      html: html,
+      iconSize: [30, 30],
+      iconAnchor: [15, 15],
+    });
+  }
+
+  korokDetails() {
```

Comment:
showKorokMarkers (the current name makes it sound like this is *returning* Korok details)